test/amazonica/test/core.clj

```diff
@@ -8,69 +8,38 @@
-                                                  delete-tags
-                                                  describe-events
-                                                  describe-tags]]
-        [amazonica.aws.simpledb         :exclude [create-domain
-                                                  delete-domain]]
-        [amazonica.aws.sns              :exclude [add-permission
-                                                  remove-permission]]
-        [amazonica.aws.storagegateway   :exclude [adjust-client-configuration
-                                                  create-snapshot
-                                                  delete-volume]]
-        [amazonica.aws.glacier          :exclude [abort-multipart-upload
-                                                  adjust-client-configuration
-                                                  complete-multipart-upload
-                                                  initiate-multipart-upload
-                                                  list-multipart-uploads
-                                                  list-parts]]
-        [amazonica.aws.opsworks         :exclude [adjust-client-configuration
-                                                  create-stack
-                                                  delete-stack
-                                                  describe-instances
-                                                  describe-stacks
-                                                  describe-volumes
-                                                  update-stack]]
-        [amazonica.aws.elasticmapreduce :exclude [add-tags
-                                                  adjust-client-configuration
-                                                  remove-tags]]
-        [amazonica.aws.ec2              :exclude [describe-tags]]
-        [amazonica.aws
-          cloudformation
-          cloudfront
-          cloudsearch
-          cloudwatch
-          dynamodbv2
-          elasticloadbalancing
-          identitymanagement
-          route53
-          s3
-          securitytoken
-          simpleemail
-          sqs]))
+  (:require [clojure.test]
+            [clojure.pprint]
+            [clojure.java.shell]
+            [amazonica.core]
+            [amazonica.aws
+             autoscaling
+             cloudformation
+             cloudfront
+             cloudsearch
+             cloudwatch
+             datapipeline
+             dynamodbv2
+             ec2
+             ecs
+             elasticache
+             elasticbeanstalk
+             elasticloadbalancing
+             elasticmapreduce
+             glacier
+             identitymanagement
+             opsworks
+             rds
+             redshift
+             route53
+             s3
+             securitytoken
+             simpledb
+             simpleemail
+             sns
+             sqs
+             storagegateway
+             ]))
 
```

Comment:
Ahh, I missed the change to require from use. I think I originally did this just to document when namespaces had conflicting function names, in case someone was `use`ing multiple namespaces.